salt-blackout-formula/metadata/form.yml

```diff
@@ -0,0 +1,36 @@
+# This file is part of blackout-formula.
+#
+# Foobar is free software: you can redistribute it and/or modify
```

Comment:
Too many foobar in the headers. Here and in the other files.

Reply:
Thanks. Removed headers.